about.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Africa Agriculture Summit|2023</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Icons+Outlined" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <script src="./js/index.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <div class="desktop-nav">
+        <div class="face-twitt">
+          <img src="./images/facebook-512.png" alt="facebook" class="facebook">
+          <img src="./images/twitter-512.png" alt="twitter" class="twitter">
+        </div>
+        <ul class="english">
+          <li><a href="#">English</a></li>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      </div>
+      <div class="desktop-items">
+        <a href="index.html"><img src="./images/LogoMakr-45qqgi.png" alt="" class="desktop-logo"></a>
+        <ul>
+          <li><a href="about.html" class="about">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Sponsors</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#" class="campaign">AA Campaign</a></li>
+        </ul>
+      </div>
+      <nav class="hidden">
+        <img src="./images/cancel.png" alt="" class="cancel">
+        <ul>
+          <li><a href="#">Home</a></li>
```

Comment:
- [Optional] Please add `index.html` as the value to the `href` attribute so that users can easily navigate to the home page on the mobile view.